app/src/main/java/movierating/repositories/MovieRepository.java

```diff
@@ -3,14 +3,15 @@
 import movierating.models.Movie;
 import movierating.models.User;
 
+import java.util.ArrayList;
 import java.util.List;
 
 public class MovieRepository {
 
-    private final List<Movie> movieList;
+    private List<Movie> movieList;
```

Comment:
Why is this a List not a Map<String, movie>?
Because I see the usage patterns are of the type get the movie object by name.

Reply:
Same here, Changed it to HashMap.